### PR TITLE
Added checkTest tests

### LINKED ISSUE #54: testing checkTest
As a developer, I want to test if my function for testing works.

### PR DESCRIPTION
**Pull request by veikkahonkanen**
Resolves #54

### CODE REVIEW

**Review by eetupur — APPROVED** [review]

File: test/checkTest.spec.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import Vorpal from "vorpal";
+import * as checkTest from "../src/commands/checkTest";
+import { CheckUtils } from "../src/classes/check-utils";
+import { CheckSet } from "../src/interfaces/types";
+
+it('exports itself', () => {
+  const vorpal = new Vorpal();
```
Comment (anttileppa) [inline test/checkTest.spec.ts:7]: What does this actually test?

File: test/checkTest.spec.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  vorpal.execSync = jest.fn();
+  vorpal.use(checkTest.test).execSync("test");
+  vorpal.execSync("exit");
+
+  expect((vorpal.use(checkTest.test).execSync)).toReturn();
+  expect((vorpal.execSync)).toReturn();
+});
+
+it('checks for prequisites', async () => {
+  const test : CheckSet[] = [{checkable : "git", details : {}}]
```
Comment (anttileppa) [inline test/checkTest.spec.ts:18]: What does this actually test?

**Review by anttileppa — CHANGES_REQUESTED** [review]

File: test/checkTest.spec.ts (hunk 1 above)
Comment (veikkahonkanen) [reply test/checkTest.spec.ts:7]: Accessibility, and its exported functionality for Vorpal. In this case, function Action(). CheckTest is currently a stub function that will run checks for projects in the future. The .spec will have more functionality in another issue.

File: test/checkTest.spec.ts (hunk 2 above)
Comment (veikkahonkanen) [reply test/checkTest.spec.ts:18]: CheckTest is currently a stub function that will run checks for projects in the future. As CheckUtils class will have more functionality in the future, the .spec will have more functionality in another issue as well.

**Pull request merged**